### ISSUE TITLE
Cannot run v1.4

### ISSUE THREAD

**Issue by arafato** [serverless/serverless#319]
Hello, 
I'm tying to run v1.4 to try out new commands such as 'jaws lambda deploy'. However, whenever I try to run './bin/jaws' I get the following error message (Ubuntu 15.10 and Mac OS X)

Error: Cannot find module '/home/arafato/repos/JAWS/lib/defaults/actions/CodeUploadLambdaNodejs.js'

All files are present, any idea on how to get this running? Thanks!

**Comment by arafato** [comment]
Got it: The file is checked in as 'CodeUploadLambdaNodeJs.js'
                                                                                                ^
